recipe/meta.yaml

```diff
@@ -47,7 +47,9 @@ outputs:
         - scipy >=0.14
         - filelock
         - libpython >=2.0  # [win]
```

Comment:
Is libpython still required here? Maybe also move this down to `aesara`?

Reply:
Good question...

Oh... I just checked, and the dependencies in general seem to have not been updated in a long time! :joy:

As I mentioned a few weeks ago (I think regarding the PyMC feedstock), the Conda-Forge infrastructure doesn't automatically update the dependencies (yet), because this is an extremely difficult problem.

I'll have a look...

Reply:
Dependencies updated, and libpython is now gone, so marking as resolved.